### ISSUE TITLE
Trying to access array offset on value of type bool in vendor/colinmollenhour/cache-backend-file/File.php on line 149

### ISSUE THREAD

**Issue by hostep** [colinmollenhour/Cm_Cache_Backend_File#37]
Hi!

Using:
- PHP 7.4.33
- Magento 2.4.3-p3 (_Update_, we moved this shop to Magento 2.4.5-p1 and PHP 8.1 and it still happened after yet another server crash)
- colinmollenhour/cache-backend-file 1.4.5

I noticed this error:
```
[2022-11-19 12:39:58] .CRITICAL: Notice: Trying to access array offset on value of type bool in vendor/colinmollenhour/cache-backend-file/File.php on line 149 [] []
```

This happened shortly after the hosting brought back a server to live, after it had crashed. So possibly some cached data could have been corrupted or something (this is just guessing).

Flushing the Magento cache stopped it from occurring.

This code probably needs more defensive code to prevent this warning and to do the right thing to make sure the cache is no longer corrupted?

**Comment by colinmollenhour** [comment]
Thanks for the report, fixed and a new release published (1.4.6)

**Comment by hostep** [comment]
Awesome, thanks Colin!
